colour/colorimetry/spectrum.py

```diff
@@ -53,6 +56,88 @@
            'zeros_spd',
            'ones_spd']
 
+DEFAULT_WAVELENGTH_DECIMALS = 10
+"""
+Default wavelength precision decimals.
+
+DEFAULT_WAVELENGTH_DECIMALS : int
+"""
+
+
+class SpectralMapping(ArbitraryPrecisionMapping):
+    """
+    Defines the base mapping for spectral data.
+
+    It enables usage of floating point wavelengths as keys by rounding them at
+    a specfic decimals count.
+
+    Parameters
+    ----------
+    data : dict or SpectralMapping, optional
+        Spectral data in a *dict* or *SpectralMapping* as follows:
+        {wavelength :math:`\lambda_{i}`: value,
+        wavelength :math:`\lambda_{i+1}`,
+        ...,
+        wavelength :math:`\lambda_{i+n}`}
```

Comment:
Argh and I just misinterpreted the LaTex notation. With `i` being the index and `\lambda_{i}` being the variable it should be fine as is.